service/scaladsl/server/src/main/scala/com/lightbend/lagom/scaladsl/server/LagomServer.scala

```diff
@@ -48,7 +48,7 @@ object LagomServer {
   }
 }
 
-trait LagomServerComponents {
+trait LagomServerComponents extends MetricsServiceComponents {
```

Comment:
A Server should have the `metricsBinding` available so users (or Lagom itself) may inject it.

`MetricsServiceComonents` provides only the `metricsBinding` and uses the `circuitBreakerMetricsProvider` that's available via `LagomServiceClientComponents` (which must be available for any LagomServiceApplication).